### PR TITLE
update cinnabar style to 7.0.1

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes part of #199 (display barriers that are important during survey)

### CODE REVIEW

File: app/src/main/assets/cinnabar-style-7.0.1.yaml [hunk 1]
```diff
@@ -718,8 +718,12 @@ textures:
 
 sources:
     mapzen:
-        type: MVT
-        url:  http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.mvt
+        type: TopoJSON #GeoJSON, MVT, TopoJSON
+        url:  https://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.topojson
+        #url: '//tile.dev.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.topojson'
+        #url:  //localhost:8080/vector/all/{z}/{x}/{y}.json
```
Comment (westnordost) [inline app/src/main/assets/cinnabar-style-7.0.1.yaml]: This must stay MVT!

Comment (matkoniecz) [inline app/src/main/assets/cinnabar-style-7.0.1.yaml]: Interesting. I missed it, but I tested it and it works nicely (at my phone at least)

Comment (westnordost) [inline app/src/main/assets/cinnabar-style-7.0.1.yaml]: Yes, it accepts any of these formats. But the binary MVT is smaller, this makes a difference with this many users.

Comment (matkoniecz) [inline app/src/main/assets/cinnabar-style-7.0.1.yaml]: Switched back to MVT, it should be now ready.

**Comment by westnordost** [comment]
cool, thanks

**Comment by westnordost** [comment]
https://github.com/westnordost/StreetComplete/commit/f4abb9762cf0eedcbebf70e9a2190696d47e8f58#commitcomment-22538246

Could you please change the url to the mvt url as well and use http (not https)?

**Comment by matkoniecz** [comment]
done and documented in #328

**Pull request merged**